src/main/java/com/snowflake/kafka/connector/internal/telemetry/SnowflakeTelemetryServiceV1.java

```diff
@@ -26,6 +26,7 @@
 public class SnowflakeTelemetryServiceV1 extends EnableLogging
     implements SnowflakeTelemetryService {
   private static final ObjectMapper MAPPER = new ObjectMapper();
+  private static final long INSERTROWS_MIN_BATCH_SIZE = 500;
```

Comment:
This was a random number i decided, but its probably too low / sends too many events to the telemetry service. Open to higher numbers

Reply:
Batching on a timeframe may be better (number of events emitted every 5 min, etc)